module.c

```diff
@@ -319,6 +308,22 @@ static PyObject *context_module(ContextData *self, PyObject *args) {
 	return context_eval_internal(self, args, JS_EVAL_TYPE_MODULE);
 }
 
+// _quickjs.Context.parse_json
+//
+// Evaluates a Python string as JSON and returns the result as a Python object. Will
+// return _quickjs.Object for complex types (other than e.g. str, int).
```

Comment:
What happens on parse failure?

Reply:
There will be a Python exception. I have added a test that shows this.